web/src/components/core/Super/SuperDeletePvc.jsx

```diff
@@ -0,0 +1,126 @@
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+    'flexGrow': 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.colors.orange,
+    width: 200,
+  },
+  margin: {
+    margin: theme.spacing(1),
+  },
+  extendedIcon: {
+    marginRight: theme.spacing(1),
+  },
+}));
+
+export default function SuperDeletePvc({id}) {
+  const [confirm, setConfirm] = useState(false);
+  const [open, setOpen] = useState(false);
+  const {enqueueSnackbar} = useSnackbar();
+  const theme = useTheme();
+  const fullScreen = useMediaQuery(theme.breakpoints.down('md'));
+
+  const deleteThePvc = () => {
+    axios.delete(`/api/super/students/pvc/${id}`).then((response) => {
+      const data = standardStatusHandler(response, enqueueSnackbar);
+    }).catch(standardErrorHandler(enqueueSnackbar));
+  };
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Box>
+      <Dialog
+        fullScreen={fullScreen}
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="responsive-dialog-title"
+      >
+        <DialogTitle id="responsive-dialog-title">
+          Are you sure you want to continue?
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText>
+            You are about to delete your Anubis Cloud IDE volume for user with ID: {`${id}`}. This will delete
+            this user's home directory, and any work that resides there. If they have a IDE
+            open, the delete will occur after their IDE is stopped. This action cannot
+            be undone. Please confirm to continue.
+          </DialogContentText>
+          <FormControlLabel
+            control={
+              <Switch
+                checked={confirm}
+                color={'error'}
+                onChange={(_, v) => setConfirm(v)}
+              />
+            }
+            labelPlacement={'end'}
+            label={<i>Yes I confirm I would like to delete this user's Cloud IDE Volume</i>}
+          />
+        </DialogContent>
+        <DialogActions>
+          <Button
+            autoFocus
+            onClick={handleClose}
+            color={'primary'}
+            variant={'contained'}
+          >
+            Cancel
+          </Button>
+          <Button
+            startIcon={<DeleteForever/>}
+            disabled={!confirm}
+            onClick={() => {
+              handleClose();
+              deleteThePvc();
+            }}
+            color={'error'}
+            variant={'contained'}
+          >
+            Delete
+          </Button>
+        </DialogActions>
+      </Dialog>
+      <Box sx={{m: 1}}>
+        <Button
+          color={'error'}
+          variant={'contained'}
+          startIcon={<DeleteForever/>}
+          onClick={handleOpen}
+        >
+          Delete
+        </Button>
+      </Box>
+    </Box>
+  );
```

Comment:
If you can attach some screenshots of your UI elements.

Reply:
Also, if we already have a component for this. It would be easier to just create a variation within the same component, instead of having two that do basically the same thing.

Reply:
Done, please see description for screenshots